dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/RootConverter.java

```diff
@@ -31,7 +32,8 @@ public RootRest convert(HttpServletRequest request) {
         String requestUrl = request.getRequestURL().toString();
         String dspaceUrl = configurationService.getProperty("dspace.server.url");
         String dspaceSSRUrl = configurationService.getProperty("dspace.server.ssr.url", dspaceUrl);
-        if (!dspaceUrl.equals(dspaceSSRUrl) && requestUrl.startsWith(dspaceSSRUrl)) {
+        if (!dspaceUrl.equals(dspaceSSRUrl) && StringUtils.isNotBlank(dspaceSSRUrl)
+            && requestUrl.startsWith(dspaceSSRUrl)) {
```

Comment:
This change doesn't seem related to OpenAlex.  I'm not against it, as it seems "logical" to verify that the request starts with the configured SSR URL.  But, I wanted to ask if this was a bug you ran into related to OpenAlex or something more general?  

The reason I wanted to ask is that **if this is a general bug**, then we may need to ensure this small fix gets **backported to 8.x and 7.6.x**.  So, it may need to be pulled out into a separate PR specific to 8.x/7.x, because we won't be backporting OpenAlex integration